server/admin/admin_server.go

```diff
@@ -39,11 +39,11 @@ func New(registry extension.Registry) *Server {
 
 // ListTrees implements trillian.TrillianAdminServer.ListTrees.
 func (s *Server) ListTrees(ctx context.Context, req *trillian.ListTreesRequest) (*trillian.ListTreesResponse, error) {
-	trees, err := s.listTreeImpls(ctx, req)
+	tl, err := s.listTreeImpls(ctx, req)
```

Comment:
Yes, package conflict. tl is treelist. I'll use resp but I don't think it's very descriptive.